### PR TITLE
An Overview of Meta-Learning

### PR DESCRIPTION
**Pull request by collins-a**
Article as index.md.
Hero image as hero.jpg.
Maml.png also included.

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by collins-a** [comment]
This fixes #888

File: articles/meta-learning/index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+**Extracting insights from Metadata.** Metadata is a set of data that describes other data. We could call it data about data. In the context of meta-learning, it is made up of knowledge of prior learning experiences. This helps in the development of a hypothesis for a new task.
+
+### Approaches to Meta-Learning Algorithms
+
+Various literature details different kinds of approaches to meta-learning. We cover three approaches in this section.
+
+#### Optimized Meta-Learning
+
+Wikipedia defines a [hyperparameter](https://en.wikipedia.org/wiki/Hyperparameter_(machine_learning)) as a parameter whose value is used to control the learning process. It is a parameter that is defined before the start of the learning process. Hyperparameters have a direct impact on the quality of the training process. Hyperparameters can be tuned. An example of a hyperparameter is the number of branches in a decision tree.
```
Comment (lalith1403) [inline articles/meta-learning/index.md:37]: Need not say Wikipedia defines hyperparameter. The definition of hyperparameter is a standard one.

File: articles/meta-learning/index.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+An example of few-shot meta-learning is the use of [memory-augmented neural networks](http://proceedings.mlr.press/v48/santoro16.pdf).
+
+[Gradient descent](https://ml-cheatsheet.readthedocs.io/en/latest/gradient_descent.html#:~:text=Gradient%20Descent.%20Gradient%20descent%20is%20an%20optimization%20algorithm,descent%20to%20update%20the%20parameters%20of%20our%20model.) is an optimization algorithm by nature. It minimizes a given function by moving towards the direction of the steepest descent iteratively. Gradient descent is used to update the parameters of a model. Traditional gradient descent networks need lots of data to learn. The training process is extensive and iterative. The models have to learn their parameters again to quickly add new information when exposed to new data. This is a very inefficient process.
```
Comment (lalith1403) [inline articles/meta-learning/index.md:59]: I don't think GD is a nature-based optimization algorithm. Ant-colonisation, PSO, etc are nature inspired.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Amazing article @collins-a. Few minor changes required.

**Comment by collins-a** [comment]
@lalith1403 thank you, made a couple of tweaks. Cheers!

**Comment by hectorkambow** [comment]
Plagiarism check done

**Pull request merged**